app/services/messages/service.go

```diff
@@ -391,13 +387,14 @@ func (ss *Service) VerifyEthereumTxAuthenticity(tm encoding.TopicMessage) (bool,
 
 	if dbTx.Amount != amount ||
```

Comment:
Add `TODO` here that we should validate the `erc20Address` as-well, once the contracts are updated